documentation/docs/user_guide/inputs/meteorological_forcing.md

```diff
@@ -0,0 +1,124 @@
+
+An example input file can can be found at the [NCI THREDDS server](https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f7075_4625_2374_0846).
+
+## Model configuration and grid
+
+Site or regional/global simulations can use either an x-y grid input file or a land only compressed grid-type input file.
+
+Meteorological variables that conform to the ALMA standard can have:
+
+- an x-y grid with 3 dimensions (x,y,t) or 4 dimensions (x,y,z,t), or
+- a compressed land-only grid of 2 dimensions (land, t)
+
+For the x-y grid, an `x` and a `y` dimension variable must be present, even if the simulation is only a single site/gridpoint. Additionally, single precision variables named `latitude` and `longitude` (or `nav_lat` and `nav_lon` if using ALMA formatting), both dependent on the x and y dimensions only, must be present. Both sea and land points may be included by using an integer `mask(x,y)` variable; a value of 1 implies a land gridpoint, anything else is assumed to be ocean.
+
+For the single dimension land-only “compression by gathering” grid (see [here](http://www.lmd.jussieu.fr/~polcher/ALMA/dataformats.html)), a single spatial dimension is used.
+An example of the NetCDF header from such a file is shown below:
+
+```
+dimensions:
+    tstep = UNLIMITED ; // (1461 currently)
+    land = 15238 ;
+    y = 180 ;
+    x = 360 ;
+variables:
+    float SWdown(tstep, land) ;
+        SWdown:axis = "TYX" ;
+        SWdown:units = "W/m^2" ;
+        SWdown:long_name="Surface incident shortwave radiation" ;
+        SWdown:associate = "time (nav_lat nav_lon)" ;
+        SWdown:missing_value = 1.e+20f ;
+    int land(land) ;
+        land:compress = "y x" ;
+    float nav_lat(y, x) ;
+        nav_lat:units = "degrees_north" ;
+        nav_lat:valid_min = -90.f ;
+        nav_lat:valid_max = 90.f ;
+        nav_lat:long_name = "Latitude" ;
+    float nav_lon(y, x) ;
+        nav_lon:units = "degrees_east" ;
+        nav_lon:valid_min = -180.f ;
+        nav_lon:valid_max = 180.f ;
+        nav_lon:long_name = "Longitude" ;
+    float time(tstep) ;
+        time:units = "seconds since 1949-01-01 00:00:00" ;
+        time:title = "Time" ;
+        time:long_name = "Time axis" ;
+        time:time_origin = " 1949-JAN-01 00:00:00" ;
+```
+
+Most of the variables will be structured as SWdown is above. The relationship between “land” and “x” and “y” inside the CABLE netcdf driver is:
+
+```fortran
+y = INT((landGrid(j)-1)/xdimsize)
+x = landGrid(j) - y*xdimsize
+y = y + 1
+```
+
+Note that the number of points simulated by offline CABLE is not specified in the code, but determined here using the number of land points found in the met file. If a patch dimension exists, the `patchfrac` variable gives the fraction of each vegetation patch in each land grid point. CABLE counts the number of "active" patches (patches with non-zero `patchfrac`) as the number of points for simulation.
```

Comment:
```suggestion
```
I'd move one part at the top and the `patchfrac` description in the variables tables.